src/app/services/blog.service.ts

```diff
@@ -6,41 +6,49 @@ export class BlogService {
   butter = Butter('2c11ed1f264363ff0e0b87e0e7f30058a444fac8');
   constructor() { }
 
-  async getPosts() {
+  async postsRef() {
```

Comment:
getPosts() is right in this case since we're just fetching it once. Same for the similar methods in this service and the Butter-based methods in CasesService.